single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/styles/iidm/LimitHighlightStyleProvider.java

```diff
@@ -0,0 +1,89 @@
+package com.powsybl.sld.svg.styles.iidm;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.sld.library.ComponentLibrary;
+import com.powsybl.sld.model.graphs.Graph;
+import com.powsybl.sld.model.graphs.VoltageLevelGraph;
+import com.powsybl.sld.model.nodes.*;
+import com.powsybl.sld.model.nodes.feeders.FeederWithSides;
+import com.powsybl.sld.svg.styles.EmptyStyleProvider;
+import java.util.*;
+
+import static com.powsybl.sld.svg.styles.StyleClassConstants.*;
+
+/**
+ * @author Jamal KHEYYAD {@literal <jamal.kheyyad at rte-international.com>}
+ */
+public class LimitHighlightStyleProvider extends EmptyStyleProvider {
```

Comment:
`OverloadHighlightStyleProvider`?

Reply:
done

Reply:
woops sorry I wanted to remove that comment. The previous name was more consistent, I forgot about the busbars when I started to review